src/App.tsx

```diff
@@ -1,24 +1,220 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import {
+  getTodos,
+  postTodo,
+  removeTodo,
+} from './api/todos';
+import { Footer } from './components/Footer';
+import { Form } from './components/Form';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 6101;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
```

Comment:
not really good idea to use strings this way, it is really easy to make a typo and it will not work as expected. Let's create a enum for filterBy